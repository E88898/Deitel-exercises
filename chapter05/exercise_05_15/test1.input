a
a
a
c
c
b
b

