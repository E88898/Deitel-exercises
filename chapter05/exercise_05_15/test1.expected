Welcome to the grade book for
CS101 C++ Programming!

Enter the letter grades.
Enter the EOF character to end input.


Number of students who received each letter grade:
A: 3
B: 2
C: 2
D: 0
F: 0
Grade-point average for the set of grades 3
